README.md

```diff
@@ -37,8 +37,8 @@ See [AEM Sites Core Components](https://docs.adobe.com/content/help/en/experienc
 The latest version of the AEM Forms Core Components, require the below minimum system requirements:
 
 | Forms Core Components | WCM Core Components | AEM Forms as a cloud service | Java  | Maven  |  
-| --------------------- | ------------------- | ---------------------------- | ----- | ------ | 
-| 1.0.16                | 2.20.2              | Continual                    | 8, 11 | 3.3.9+ |
+| --------------------- |---------------------| ---------------------------- | ----- | ------ | 
+| 1.0.16                | 2.21.0              | Continual                    | 8, 11 | 3.3.9+ |
```

Comment:
we should not update min system requirement if we are not using any new api from wcm.